plugins/modules/content_view.py

```diff
@@ -68,6 +68,11 @@
     description:
       - Solve RPM dependencies by default on Content View publish
     type: bool
+  import_only:
+    description:
+      - Designate this Content View for importing from upstream servers only.
+    type: bool
+    version_added: 3.14.0
```

Comment:
What's the point of `version_added`?

Reply:
It gets rendered in the docs, so the reader can know since which version they are able to use this specific parameter without checking all the docs/changelig versions

Reply:
See https://theforeman.github.io/foreman-ansible-modules/develop/plugins/content_view_module.html#parameter-import_only